bibedit/record_conference_paper.xml

```diff
@@ -23,6 +23,10 @@ along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
     <subfield code="a">VOLATILE:e.g 10.1103/example.doi.for.inspire</subfield>
     <subfield code="2">VOLATILE:DOI</subfield>
   </datafield>
+  <datafield tag="035" ind1=" " ind2=" ">
+    <subfield code="a">VOLATILE:external record identifier</subfield>
+    <subfield code="9">VOLATILE:source</subfield>
```

Comment:
What are the known sources so far? Are they enumerable? (thinking about when we move this to JSONSchema)